subiquity/server/controllers/mirror.py

```diff
@@ -280,13 +280,9 @@ def on_geoip(self):
             self.model.set_country(self.app.geoip.countrycode)
         self.cc_event.set()
 
-    def on_source(self):
-        # FIXME disabled until we can sort out umount
```

Comment:
we're giving up on this ?

Reply:
It was moved - https://github.com/canonical/subiquity/pull/1609/files#diff-313c6a6b896f43cafe50da31415cf91368d0a7a7b1f4fc334ac39cf9074e5644R275

Reply:
Oh, right, thanks!